docs/getting-started.rst

```diff
@@ -152,6 +168,33 @@ With ``--stop-on-failure``
 
   Result: Failed
 
+Charm mapping
+"""""""""""""
+
+This option enables user to explicitly tell ``juju-verify`` which charm a
```

Comment:
Typo: "users"

Reply:
Just out of curiosity, is it wrong to talk about user in singular here? ('cause that was my intention :D ) I can of course change it to plural.

Reply:
Singular is ok. Maybe "the user", then?